src/test/java/ca/corefacility/bioinformatics/irida/ria/integration/projects/ProjectRemoteSettingsIT.java

```diff
@@ -53,10 +53,6 @@ public void testRemoteProjectSettings() {
 		assertFalse("URL should not be empty", url.isEmpty());
 		page.submitProject();
 
-		ProjectDetailsPage projectDetailsPage = ProjectDetailsPage.initElements(driver());
```

Comment:
Does `submitProject` have a changed behaviour? It looks like previously according to this test it went to the `ProjectDetailsPage` and now it goes to the `ProjectRemoteSettingsPage`?